_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -26,7 +26,8 @@ This rule applies to any HTML `input`, `select` and `textarea` element with an `
 - **hidden**: the element is not [visible][], and not [included in the accessibility tree][]; or
 - **disabled**: the element is a [disabled element]; or
 - **fixed value**: the element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
-- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
+- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][]; or
+- **autofill expectation** the `autocomplete` attribute wears the [autofill expectation mantle](https://html.spec.whatwg.org/#autofill-expectation-mantle) that does not consist of a single token that is [ASCII case-insensitive][] match for string `off` or string `on`.
```

Comment:
I think we can get rid of the "mantle" bit (which is a bit tricky…)
The "anchor" mantle is only worn on `input` elements with a `type` attribute in the `Hidden` state, i.e a `type` attribute value of `hidden`. Hence, they are already excluded by the **fixed value** condition.
```suggestion
- **autofill expectation**: the `autocomplete` attribute consists of a single token that is [ASCII case-insensitive][] match for string `off` or string `on`.
```

Given that the Applicability starts with "with an autocomplete attribute value that is neither empty ("") nor only ASCII whitespace", we can probably refactor the full list as:
```
This rule applies to any HTML input, select and textarea element with an autocomplete attribute, except if one of the following is true:

- **autofill expectation**: the autocomplete attribute it either empty (""), only ASCI whitespace, or consists of a single token that is ASCII case-insensitive match for the string "off" or "on"; or
- **hidden*: the element is not visible, and not included in the accessibility tree; or
- **disabled**: the element is a disabled element; or
- **fixed value**: the element is an input element with a type attribute value of either hidden, button, submit or reset; or
- **static**: the element is not part of sequential focus navigation and has a semantic role that is not a widget role.
```